_includes/schemas/shipments_response.json

```diff
@@ -2,38 +2,34 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "properties": {
-    "id": {
+    "carrier": {
       "type": "string",
-      "description": "the shipment id that can be used for requesting info about a shipment or tracking it"
+      "enum": ["dhl", "ups", "dpd", "hermes", "gls"],
+      "description": "acronym of the carrier"
     },
     "carrier_tracking_no": {
       "type": "string",
       "description": "the original tracking number that can be used on the carriers website"
     },
-    "carrier": {
-      "type": "string",
-      "enum": ["dhl", "ups", "dpd", "hermes", "gls"],
-      "description": "acronym of the carrier"
-    },
     "created_at": {
       "type": "date-time",
       "description": "timestamp the shipment was created"
     },
-    "tracking_url": {
+    "from": {
+      "$ref": "#/definitions/address",
+      "description": "If missing, the default sender address (if defined in your shipcloud account) will be used"
+    },
+    "id": {
       "type": "string",
-      "description": "URL you can send your customers so they can track this shipment"
+      "description": "the shipment id that can be used for requesting info about a shipment or tracking it"
     },
     "label_url": {
       "type": "string",
       "description": "URL where you can download the label in pdf format"
     },
-    "from": {
-      "$ref": "#/definitions/address",
-      "description": "If missing, the default sender address (if defined in your shipcloud account) will be used"
-    },
-    "to": {
-      "$ref": "#/definitions/address",
-      "description": "the receivers address"
+    "notification_email": {
+      "type": "string",
+      "description": "email address of the buyer who should be notified of a change of shipment status by shipcloud"
```

Comment:
I feel like we should use the word "receiver" instead of "buyer". What do you think?